lib/streamlit/elements/widgets/button.py

```diff
@@ -421,6 +424,30 @@ def link_button(
             use_container_width=use_container_width,
         )
 
+    @gather_metrics("page_link")
+    def page_link(
+        self,
+        page: str,
```

Comment:
nit: what can `page` be? Can it be a name, a path, a URL or all of it? If it is just a name or a path, I would call it `page_path` or `page_name` to give a hint to the user how to use it and probably leave a comment with an example of how to call this method.

Reply:
This was quite the contentious product debate 😂 
`page` is a path when used for MPA use case, or a URL for external links - it was initially called `page_path` but then they opted for `page` - I'll re-confirm though!